### DISCUSSION TITLE
Running program

### DISCUSSION THREAD

**Discussion by Rune222** [hydrogen-music/hydrogen#1361] [Hydrogen on Windows]
Just reloaded my hydrogen onto my laptop running W7,replaced with an updated version.But can't find how to install it after I've unpacked it with WinRAR...No amount of head scritching will aid this..I know sombody out there will put me out of my misery

**Comment by cme** [comment]
I don't quiite follow the desciption of what you've done, but you should install from one of the release installers. These are regular Windows EXE installers and they can be just run as normal, with no need for WinRAR to be involved at any point in the process.

https://github.com/hydrogen-music/hydrogen/releases

**Comment by Rune222** [comment]
thanks,yes your right WinRAR is the problem
